src/site/content/en/secure/what-is-mixed-content/index.md

```diff
@@ -89,13 +80,13 @@ automatic upgrading of passive mixed content where possible.
 Automatic upgrading means that if the asset is available over HTTPS, but has been hardcoded as HTTP,
 the browser will load the HTTPS version. If no secure version can be found the asset will not load.
 
-Chrome logs messages to its DevTools Console whenever it detects mixed content
-or auto-upgrades passive mixed content.
+Whenever it detects mixed content or auto-upgrades passive mixed content, Chrome logs detailed messages to the Issues Tab in DevTools
+to guide you on how to fix the specific issue.
 
 <figure class="w-figure">
   <img class="w-screenshot"
       src="passive-mixed-content.jpg"
-      alt="Chrome DevTools showing the warnings displayed when mixed content is detected and upgraded">
+      alt="The Issues tab in DevTools shows detailed information about the specific mixed content issue and how to fix it">
 </figure>
 
```

Comment:
Suggestion/Nit (not a must for this PR since the original article is like that, but while we're at it): we could add a mention of what other browsers do, in order to make this web.dev article browser-agnostic.
E.g. `Firefox and Safari log messages in the developer console.`